python/lsst/daf/butler/core/storageClass.py

```diff
@@ -342,6 +342,22 @@ def __init__(self, config=None):
         if config is not None:
             self.addFromConfig(config)
 
+    def __str__(self):
+        """Return summary of factory.
+
+        Returns
+        -------
+        summary : `str`
+            Summary of the factory status.
+        """
+        sep = "\n"
+        return f"""Number of registered StorageClasses: {len(self._storageClasses)}
+
+StorageClasses
+--------------
+{sep.join(f"{s}: {self._storageClasses[s]}" for s in self._storageClasses)}
+"""
+
```

Comment:
This would be nicer if the text was in a similar formatting, so maybe:
```
        returnString = f"""
            Number of registered StorageClasses: {len(self._storageClasses)}
            StorageClasses
            --------------
            {sep.join(f"{s}: {self._storageClasses[s]}" for s in self._storageClasses)}
             """
        return textwrap.dedent(returnString.lstrip("\n"))   
```

Reply:
I don't really like all the dedent shenanigans and I don't have a problem with the left justification so I'd prefer to leave it as is.